src/App.js

```diff
@@ -2,10 +2,65 @@ import React from "react";
 import "./App.scss";
 
 function App() {
-  return (
-    <div className="container">
-    </div>
-  );
+    return (
+        <div>
+            <div className="section">
+                <div className="columns">
+                    <div className="column is-3">
+                        <section className="hero header">
+                            <div className="hero-body">
+                                <div className="container">
+                                    <img src={"/images/unicorn.svg"} className="logo image is-48x48"/>
+                                    <h1 className="title">
+                                        calc.sg
+                                    </h1>
+                                    <h2 className="subtitle">
+                                        insert subtitle here
+                                    </h2>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                    <div className="column">
+                        {HomepageLayout()}
```

Comment:
Why not use a react component?
```suggestion
                        <Homepage />
```